### PR TITLE
fix(table): Allow setting of `title: ""` to hide the default cell tooltip

### PR DESCRIPTION
**Pull request by stanislavgeorgiev**
Closes IBM/carbon-components-angular#1914

#### Changelog

**Changed**

* Changed comparison to be aware of empty strings

### CODE REVIEW

**Review by zvonimirfras — APPROVED** [review]
Makes sense, thanks!

**Comment by stanislavgeorgiev** [comment]
Thanks @zvonimirfras. The PR merge is blocked `Merging can be performed automatically with 2 approving reviews.`

**Comment by zvonimirfras** [comment]
> Thanks @zvonimirfras. The PR merge is blocked `Merging can be performed automatically with 2 approving reviews.`

Yes, it's waiting for @Donisius approval.

**Comment by stanislavgeorgiev** [comment]
Ah OK, I saw there's only one PR reviewer and was wondering what's happening

**Review by Donisius — APPROVED** [review]

**Pull request merged**
